NOTICE

```diff
@@ -6,3 +6,9 @@ The Autoware Foundation (https://www.autoware.org/).
 
 This product includes code developed by TIER IV.
 Copyright 2017 TIER IV, Inc.
+
+This product includes code developed by AutoCore.
+Copyright 2022 AutoCore
```

Comment:
```diff
- Copyright 2022 AutoCore
+ Copyright 2022 AutoCore Technology (Nanjing) Co., Ltd.
```

Reply:
```suggestion
Copyright 2022 AutoCore Technology (Nanjing) Co., Ltd.
```

Reply:
Updated in 65515f368493a316f8c32a11e5d39939226ec08c.